docs/language/package/pkgtutorial.mdx

```diff
@@ -315,7 +315,7 @@ It assumes that pin 23 is connected to an LED (and an appropriate resistor).
 LED-PIN ::= 23
 
 main:
-  pin := gpio.Pin.out LED-PIN
+  pin := gpio.Pin --output LED_PIN
```

Comment:
```suggestion
  pin := gpio.Pin --output LED-PIN
```

Reply:
A follow-up PR runs kebabify over the code.